concordium-consensus/src/Concordium/GlobalState/Persistent/Account/StructureV1.hs

```diff
@@ -994,13 +1162,35 @@ addBakerV1 binfo stake restake acc = do
               accountEnduringData = newEnduring
             }
 
+-- | Remove a baker/delegator from an account.
+--  This removes any baker or delegator record and sets the active stake to 0.
+--  This does not affect the stake in cooldown, which should be updated separately.
+removeStake ::
+    ( MonadBlobStore m,
+      IsAccountVersion av,
+      AccountStructureVersionFor av ~ 'AccountStructureV1,
+      AVSupportsFlexibleCooldown av
+    ) =>
+    -- | Account remove staking from.
```

Comment:
typo?

Reply:
Yes, but I just realized that this does the same as `removeStaking` below, so I think we should remove this function and just use `removeStaking` instead.

Reply:
Removing this.